src/codeu/chat/server/user_recommendation/K_Means.java

```diff
@@ -0,0 +1,209 @@
+package codeu.chat.server.user_recommendation;
+
+import codeu.chat.common.Message;
+import codeu.chat.common.SQLFormatter;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import codeu.chat.server.user_recommendation.UserFeatures;
+import codeu.chat.server.user_recommendation.Cluster;
+import codeu.chat.server.user_recommendation.Mood;
+import codeu.chat.server.user_recommendation.moodClassifier;
+import codeu.chat.server.Model;
+
+import java.io.*;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.*;
+
+import opennlp.tools.postag.POSModel;
+import opennlp.tools.postag.POSSample;
+import opennlp.tools.postag.POSTaggerME;
+import opennlp.tools.tokenize.WhitespaceTokenizer;
+
+import javax.swing.text.html.HTMLDocument;
+
+/**
+ * Created by strobe on 23/05/17.
+ */
+public class K_Means {
+
+  private Vector<UserFeatures> userVector;
+  private Vector<Cluster> clusterVector;
+  private Collection<String> allKeyWords;
+
+  private final Model model;
+  private moodClassifier classifier;
+
+  public K_Means(Model model) {
+
+    this.model = model;
+    this.classifier = new moodClassifier();
+    //Set the Model File to a correct location after build
+  }
+
+  private String Tagger(String sentence) throws Exception{
+    String taggedSentence = null;
+    InputStream inputStream = new FileInputStream("./bin/codeu/chat/server/user_recommendation/tools/en-pos-maxent.bin");
+    POSModel posModel = new POSModel(inputStream);
+
+    POSTaggerME tagger = new POSTaggerME(posModel);
+
+    WhitespaceTokenizer whitespaceTokenizer = WhitespaceTokenizer.INSTANCE;
+    String[] tokens = whitespaceTokenizer.tokenize(sentence);
+
+    String[] tags = tagger.tag(tokens);
+
+    POSSample sample = new POSSample(tokens, tags);
+
+    taggedSentence = sample.toString();
+
+    return taggedSentence;
+  }
+
+  private void InitializeClusters() {
```

Comment:
Make first char lower case here and elseqhere to be consistent with other function names.